config.js

```diff
@@ -495,6 +495,9 @@ var config = {
     // Hides add breakout room button
     // hideAddRoomButton: false,
 
+    // Require users to always specify a display name.
```

Comment:
This config could use a better description.

*(P.S. Thanks for this! Much cleaner than just disabling the input field using readOnlyName option.)*

Reply:
updated, thank you